ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/logger/SizeRotatingStoreFileListener.java

```diff
@@ -48,7 +48,10 @@ private void reset()
     @Override
     public synchronized void onAcquired(int cycle, File file)
     {
-        LOG.debug("Chronicle acquired [{}] {} at {} bytes", cycle, file.getPath(), file.length());
+        if(LOG.isDebugEnabled())
```

Comment:
Is it really necessary to do all these .isDebugEnabled? Isn't LOG.debug already handling that?

Reply:
Isn't the idea to avoid handling the arguments that are sent to the logger even if debug is disabled. That's why its only on log entries that has arguments.

Reply:
Depends on the actual logger (and version) used. Some loggers parse arguments passed lazily. That is, nothing is really done unless debugging is enabled.

Reply:
But you still have to execute file.getPath() and file.length() to get the values to pass as arguments to the logger.